README.md

```diff
@@ -6,3 +6,28 @@ Various Modelling to Generate Alternative schemes for different energy system op
 | Algorithm | PyPSA + linopy |
 |-----------|----------------|
 |[SPORES](https://doi.org/10.1016/j.joule.2020.08.002) | ✅ |
+
+## Running MGA4all
+
+We separate out the modelling backends into optional groups
+(e.g. `pypsa`).  So depending on the model someone wants to work with,
+you have to choose the appropriate backend.
+
+We prefer using `hatch` to create/manage necessary environments and
```

Comment:
I don't expect most people to be familiar with `hatch` at the moment (anecdotal evidence: I'd never used it until testing this), so I'd suggest at least adding a link to the installation instructions:

```suggestion
We prefer using [`hatch`](https://hatch.pypa.io/latest/install/) to create/manage necessary environments and
```